src/components/CustomRoute/CustomRoute.js

```diff
@@ -45,7 +45,7 @@ CustomRoute.propTypes = {
         writeAccess: PropTypes.bool
     }).isRequired,
     componentProps: PropTypes.any,
-    Component: PropTypes.func.isRequired
+    Component: PropTypes.oneOfType([PropTypes.func, PropTypes.object]).isRequired
```

Comment:
I dont think this should be `object`. Did you try `element`?

Reply:
I think it should be an object because it gets an object, when the `Component` is not loaded yet. PropWarning starts to appear after I used lazy imports. Element throws an error.